ui/app/registration/controllers/editPatientController.js

```diff
@@ -10,10 +10,36 @@ angular.module('bahmni.registration')
             $scope.patient = {};
             $scope.actions = {};
             $scope.myForms = {};
+            $scope.NID = {};
             $scope.addressHierarchyConfigs = appService.getAppDescriptor().getConfigValue("addressHierarchy");
             $scope.disablePhotoCapture = appService.getAppDescriptor().getConfigValue("disablePhotoCapture");
             $scope.today = dateUtil.getDateWithoutTime(dateUtil.now());
             $rootScope.isEligibleForVisit = true;
+
+            var splitNID = function () {
```

Comment:
@simaomazive why are you creating new variables here for codeOfHealthFacility instead of initializing the `$scope.NID` object ?

Reply:
The new variables I created use in **editpatient.html**